client/my-sites/checkout/checkout-thank-you/pending/index.tsx

```diff
@@ -188,6 +186,19 @@ function useRedirectOnTransactionSuccess( {
 	const willAutoRenew = firstPurchase?.willAutoRenew ?? false;
 	const saasRedirectUrl = getSaaSProductRedirectUrl( receipt );
 
+	const { searchParams } = getUrlParts( redirectTo || '/' );
+	const isConnectAfterCheckoutFlow =
+		Boolean( searchParams.size ) &&
```

Comment:
minor nitpick: `searchParams.size` is falsy unless it has an actual value, right?! I see no problem in removing the type conversion here then

Reply:
Yes your right, good point. 👍 🙂